src/main/java/io/qdrant/client/ExpressionFactory.java

```diff
@@ -0,0 +1,205 @@
+package io.qdrant.client;
+
+import io.qdrant.client.grpc.Points.Condition;
+import io.qdrant.client.grpc.Points.DecayParamsExpression;
+import io.qdrant.client.grpc.Points.DivExpression;
+import io.qdrant.client.grpc.Points.Expression;
+import io.qdrant.client.grpc.Points.GeoDistance;
+import io.qdrant.client.grpc.Points.MultExpression;
+import io.qdrant.client.grpc.Points.PowExpression;
+import io.qdrant.client.grpc.Points.SumExpression;
+
+/** Convenience methods for constructing {@link Expression} */
+public final class ExpressionFactory {
```

Comment:
I think it would be useful to have a test that uses this, at least to have an example to be able to point folks to.

Reply:
We'll have example snippets at qdrant.tech/documentation and api.qdrant.tech/

See https://github.com/qdrant/landing_page/pull/1506